sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/Constants.java

```diff
@@ -98,4 +98,9 @@
      * The duration in seconds to decline offers the scheduler does not need for a short time.
      */
     public static final int SHORT_DECLINE_SECONDS = 5;
+
+    /**
+     * Indent (# of spaces) to be used to pretty print json objects.
+     */
+    public static final int JSON_INDENT_FACTOR = 2;
```

Comment:
I think a good place for it would be in `ResponseUtils` since it is somewhat specific to that, with the one exception being the build info log in `SchedulerConfig`

Reply:
My bad, i thought this file was `com.mesosphere.sdk.Constants` anyway, i just resorted to use 2 as we used to do before.